andela/lotus-ah-frontend
# #160451557 Reset Password Functionality

#### What does this PR do?
     This PR implements Reset/Forgot password functionality
#### Description of Task to be completed?
      1. Configure the resetReducer
      2. Configure the reset action and add action type
      3. Add containers and components
      4. Test reducers, actions and components
#### How should this be manually tested?
      On localhost
        1. start the app on port `8080`
        2. Navigate to sign in
        3. Click on forgot password
        4. input your password 
        5. An email will be sent to you, click on reset password
        6. input password and confirm password
        7. proceed to reset your password
#### Any background context you want to provide?
        N/A
### What are the relevant pivotal tracker stories?
       #160451557
#### Screenshots (if appropriate)
<img width="1680" alt="screen shot 2018-11-13 at 1 13 36 pm" src="https://user-images.githubusercontent.com/43237059/48413042-b712f500-e746-11e8-9312-d373bc13b0e7.png">
<img width="1680" alt="screen shot 2018-11-13 at 1 13 49 pm" src="https://user-images.githubusercontent.com/43237059/48413050-c2662080-e746-11e8-85e5-4c6cd578529b.png">
<img width="1680" alt="screen shot 2018-11-13 at 1 13 58 pm" src="https://user-images.githubusercontent.com/43237059/48413066-cf830f80-e746-11e8-87d3-c18d63523347.png">
<img width="1680" alt="screen shot 2018-11-13 at 1 14 16 pm" src="https://user-images.githubusercontent.com/43237059/48413076-dc9ffe80-e746-11e8-8a50-270268be7ab8.png">

#### Questions:
        N/A
